lib/ecto/query/builder/frame_clause.ex

```diff
@@ -0,0 +1,24 @@
+import Kernel, except: [apply: 2]
+
+defmodule Ecto.Query.Builder.FrameClause do
```

Comment:
I don't think we need a separate module for this, we can check it directly in the `Ecto.Query.Builder.Windows`. We can also merge the test with the windows test.

Reply:
We should also add some tests that make sure that it works both on over and on window, probably inside `windows_test`. Before we would add those tests to the PG/MySQL unit tests, but now that they live in another repository, it means we need better coverage here. :)